pkg/kudoctl/cmd/output/output.go

```diff
@@ -0,0 +1,72 @@
+package output
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"strings"
+
+	"sigs.k8s.io/yaml"
+)
+
+const (
+	TypeYAML = "yaml"
+	TypeJSON = "json"
+)
+
+func WriteObjects(objs []interface{}, outputType string, out io.Writer) error {
```

Comment:
`WriteOjbects` doesn't call `WriteObject`? 🤔

Reply:
Yeah... Fair enough, I cleaned it up a bit more...